main/modes/games/2048/2048_game.c

```diff
@@ -255,6 +255,18 @@ void t48_gameLoop(t48_t* t48, int32_t elapsedUs)
         // Accept input again
         t48->acceptGameInput = true;
     }
+
+    if (t48->tiltControls)
+    {
+        const int indicatorx    = 190;
+        const int indicatory    = 10;
+        const int indicatorSize = 9;
+        int dx                  = CLAMP(t48->lastIMUx >> 4, -indicatorSize, indicatorSize);
```

Comment:
I love this little indicator but the axes feel inverted to me. Like I want the dot to move in the direction of the slide, so I'll do that. It may be personal preference, but I'm the person :smile: